sys/sys/intr.h

```diff
@@ -39,6 +39,8 @@
 
 #define	INTR_IRQ_INVALID	0xFFFFFFFF
 
+typedef uint64_t intr_irq_t;
```

Comment:
I wonder if this shouldn't just be `unsigned long`.

Reply:
Yeah, that or `size_t`. `uintptr_t` is definitely wrong, and `unsigned long`/`size_t` would be equivalent to what is upstream.

Reply:
Possibly rman_res_t if this value is meant to be used as the "IRQ" value in `struct resource`.